src/style/AccountPage.less

```diff
@@ -61,6 +61,25 @@ A .TubeLine div contains seberal sections with TubeLine-img and TubeLine-line
 
 .AccountPage {
 
+	.account-sidebar {
+		h5 {
+			text-transform: uppercase;
+			font-weight: 600;
+			font-family: 'Montserrat', sans-serif;
+		}
```

Comment:
@dodo721 
Avoid specific font overrides unless they are needed - in which case put a comment in to explain why.
Put the comment in at the same time as writing the rule.

Section specific rules make for bloated and fragile css -- and inconsistent styling. They are often the right thing - But the reason must be documented.